tests/unit/test_protocol_errors.py

```diff
@@ -22,6 +22,7 @@ class TestFrontendErrors(unittest.TestCase):
     """
     @classmethod
     def setUpClass(cls):
+        reload(frontend)
```

Comment:
I think reload is fine here, but we should add a comment to say why it's necessary.